wdl-format/tests/format/clays_complex_script/source.formatted.wdl

```diff
@@ -70,7 +70,8 @@ task a_task {
     # the TextMate language that it's a custom struct.
     }
 
-    command <<< >>>
+    command <<<
```

Comment:
Is this desired? I think, for a totally blank command, I'd prefer `command <<<>>>`. But that is a special case, and I'm also not set on that if you all have made a decision otherwise (please move forward in that case).

Reply:
@adthrasher has made similar comments about empty sections/tasks/workflows which all currently behave the same (a newline is always inserted after the open delim, even when the body is empty).

IMO these are corner cases not worth the added complexity to address. Granted, that added complexity isn't very great, a few lines in maybe half a dozen places. But that's still extra complexity for an edge case that I just don't think matters.

If folks really want to see empty bodies special cased, I'll add it. But my thinking is we leave as is.

Reply:
I think that's fine, @a-frantz. The additional complication would be, here, this command block isn't truly empty, it contains a single space. Should that be deleted and treated as empty? That would differ from our approach elsewhere.